news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,84 @@
+- [Fix a bad alignment of label in the email theme page](https://github.com/PrestaShop/PrestaShop/issues/26915)
+- [Fix the newsletter block overlap on customer reassurance  block](https://github.com/PrestaShop/PrestaShop/issues/26846)
+- [CI is broken on autoupgrade - PHPStan with 1.6.1.18](https://github.com/PrestaShop/PrestaShop/issues/26861)
+- [Remove duplication of cache folders in letting Symfony handle kernel reboot for tests](https://github.com/PrestaShop/PrestaShop/issues/26924)
+- [Fix untranslatable Tabs for shops upgraded to 1.7.8.0 ( BC Break)](https://github.com/PrestaShop/PrestaShop/issues/26933)
+- [Fix menu and searchbar on tablet on classic (BC Break)](https://github.com/PrestaShop/PrestaShop/issues/26643)
+- [Allow vertical scroll on product images](https://github.com/PrestaShop/PrestaShop/issues/26597)
+- [Fix product width on featuredproducts](https://github.com/PrestaShop/PrestaShop/issues/26992)
+- [Allow all characters for Cart Rule & Order Cart Rule names](https://github.com/PrestaShop/PrestaShop/issues/25829)
+- [Add an option to regenerate RTL file during the upgrade](https://github.com/PrestaShop/PrestaShop/issues/27307)
+
+
+## ## Modules 
+
+- [Excludes vendor, lib, tests directories when scanning modules' directories to extract translation catalogue](https://github.com/PrestaShop/PrestaShop/issues/24987)
+- [Fix color picker bug when multiple colorpickers are present](https://github.com/PrestaShop/PrestaShop/issues/25013)
+- [Error with dynamical form rendering in 1.7.8.1](https://github.com/PrestaShop/PrestaShop/issues/26786)
+- [Fix product width on featured products](https://github.com/PrestaShop/PrestaShop/issues/26705)
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.3).
+
+## ## Security fix 
+
+One security fix have been included in this patch version:
+
+- Server Side Twig Template Injection [security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-mrq4-7ch7-2465)
+
+More information about why it is important to update:
+
+## ## Acknowledgments
+
+Contributors to this patch version, from both the Core team and the community at large:
+
+Boubker Bribri, Daniel Hlavacek, Franck Lefèvre, Hibatallah Aouadni, Ibrahima Sow, Igor Stępień, Jonathan Lelievre, Louis Authie, M. Parvazi, Marcin Sz, Mathieu Ferment, Matthieu Rolland, okom3pom, Pablo Borowicz, Pierre Rambaud, Sharak, Thomas Baccelli, Valentin Szczupak, Web Helpers Pau
+
+Thank you!
+
+[Download PrestaShop 1.7.8.3 now!](https://www.prestashop.com/en/download){: .btn .btn-lg .btn-success}
+
+{: .text-center}
+
+Since version 1.7.8.3 is a “patch” update to version 1.7.8.1, upgrading from any 1.7.8 version will be easy: features will work better, and modules & themes which worked fine on 1.7.8.x will work just as well with 1.7.8.2. Upgrades from a standard 1.7.x version should work just as well.
```

Comment:
```suggestion
Since version 1.7.8.3 is a “patch” update to version 1.7.8.2, upgrading from any 1.7.8 version will be easy: features will work better, and modules & themes which worked fine on 1.7.8.x will work just as well with 1.7.8.2. Upgrades from a standard 1.7.x version should work just as well.
```